pkgs/build-support/dotnet/nuget-to-nix/nuget-to-nix.sh

```diff
@@ -62,28 +107,26 @@ for package in *; do
         continue
     fi
 
-    for source in "${remote_sources[@]}"; do
-      url="${base_addresses[$source]}$package/$version/$package.$version.nupkg"
-      if [[ "$source" == "$used_source" ]]; then
+    source=$(get_source_by_key "$package")
```

Comment:
This will no longer loop over sources.  I believe that would still be needed.

Reply:
I think I know how to address this and hard-coded path